src/UI/Implementation/Component/ViewControl/Renderer.php

```diff
@@ -111,9 +111,9 @@ protected function renderSectionButton(Component\Button\Button $component, $tpl,
 
 	protected function renderSortation(Component\ViewControl\Sortation $component, RendererInterface $default_renderer) {
 		$f = $this->getUIFactory();
-		$param = $component->getParameterName();
+		$component_identifier = $component->getIdentifier();
 		$options = $component->getOptions();
-		$sort_value = @$_GET[$param];
+		$sort_value = @$_GET[$component_identifier];
```

Comment:
I don't like this, because: global, silencer, magic. Let's discuss the alternatives.